src/renderers/shaders/ShaderChunk/common.glsl

```diff
@@ -4,57 +4,136 @@
 #define LOG2 1.442695
 #define EPSILON 1e-6
 
-float square( in float a ) { return a*a; }
-vec2  square( in vec2 a )  { return vec2( a.x*a.x, a.y*a.y ); }
-vec3  square( in vec3 a )  { return vec3( a.x*a.x, a.y*a.y, a.z*a.z ); }
-vec4  square( in vec4 a )  { return vec4( a.x*a.x, a.y*a.y, a.z*a.z, a.w*a.w ); }
+float square( in float a ) { return a * a; }
+vec2  square( in vec2 a )  { return vec2( a.x * a.x, a.y * a.y ); }
+vec3  square( in vec3 a )  { return vec3( a.x * a.x, a.y * a.y, a.z * a.z ); }
+vec4  square( in vec4 a )  { return vec4( a.x * a.x, a.y * a.y, a.z * a.z, a.w * a.w ); }
+
```

Comment:
BTW @benaadams if we replace the body of square with (a*a) there would be no pref loss.  The majority of all functions are inlined by GLSL shaders these days.